src/core/Table/hooks/useResizeColumns.tsx

```diff
@@ -392,7 +392,9 @@ const getPreviousResizableHeader = <T extends Record<string, unknown>>(
   headerColumn: ColumnInstance<T>,
   instance: TableInstance<T>,
 ) => {
-  const headersList = headerColumn.parent?.columns || instance.flatHeaders;
+  const headersList = (
+    headerColumn.parent?.columns || instance.flatHeaders
+  ).filter(({ isVisible }) => isVisible);
```

Comment:
![ColumnManagerResizerShown](https://user-images.githubusercontent.com/78564221/161457045-141a2b22-9144-43a7-9834-540e4186b9f5.gif)

This is to fix the resizer being shown after a column is hidden. As you can see, the change works, but doesn't refresh until another action is done to the table such as clicking on a row

Reply:
Fixed